src/main/java/org/apache/jackrabbit/filevault/maven/packaging/FormatDocviewXmlMojo.java

```diff
@@ -109,7 +111,14 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             getLog().info("Skip executing mojo 'format-xml' for incremental builds as parameter 'enableForIncrementalBuilds' is set to 'false'");
         } else {
             File jcrSourceDirectory = AbstractSourceAndMetadataPackageMojo.getFirstExistingDirectory(jcrRootSourceDirectory);
-            executeInternal(jcrSourceDirectory);
+            if (jcrSourceDirectory != null) {
+                executeInternal(jcrSourceDirectory);
```

Comment:
Let us add a null annotation to executeInternal to clearly state that null is not accepted.

Reply:
Done (squashed commits)